### DISCUSSION TITLE
Verbindung zu MEGA-Cloud

### DISCUSSION THREAD

**Discussion by dodona2408** [iterate-ch/cyberduck#15651] [Q&A] [answered]
Wird eine Verbindungsmöglichkeit zu MEGA-Cloud möglich werden?

**Comment by tatze96** [comment, marked as answer]
Es ist derzeitig nur ein SDK, aber keine öffentliche API verfügbar, die wir implementieren könnten, weshalb wir MEGA aktuell nicht unterstützen können.

Außerdem scheint es nicht ohne weiteres möglich zu sein, sich über SFTP, FTP oder WebDAV zu MEGA zu verbinden. In der MEGA Dokumentation ist angegeben, das bestimmte Ordner über WEBDAV oder FTP innerhalb MEGA CMD zugänglich gemacht werden können, ich kann aber keine Informationen darüber liefern, wie gut oder zuverlässig dies funktioniert.

Der einfachste Weg, MEGA über Cyberduck zu erreichen - wenn auch keine direkte Verbindung - wäre MEGA lokal über die Desktop-App einzubinden und dann über das "Lokal"-Protokoll auf die Dateien zuzugreifen. Das "Lokal"-Protokoll ist nach dem Namen deines Computers benannt.

---

There is only an SDK but no public API available that we could implement, which is why we are currently unable to support MEGA.

Additionally, it's not possible to connect to MEGA through SFTP, FTP, or WebDAV without further ado. It seems that you can make specific folders accessible via WebDAV or FTP through MEGA CMD, but I can't provide information on how well this works.

The easiest way to reach MEGA through Cyberduck - though not a direct connection -  would be to connect to MEGA locally through the desktop app and then access the files through the local protocol. The local or local disk protocol is named after your computer name.

**Comment by IamMille** [comment]
How about this?
https://mega.io/developers#sdk

> The MEGA SDK consists of code and documentation that enables you to make use of MEGA’s API functionality at a comfortably high level of abstraction.

**Reply by dkocher** [reply]
We need a public API, not an SDK.
